source/ftrack_connect_pipeline/constants/__init__.py

```diff
@@ -49,13 +49,16 @@
 #: Post_import component stage name.
 POST_IMPORTER = 'post_importer'
 
+
 # Common definition/client types.
 #: Opener client and its definition.
 OPENER = 'opener'
 #: Loader client and its definition used with assembler
 LOADER = 'loader'
 #: Publisher client and its definition.
 PUBLISHER = 'publisher'
+#: Batch bublisher client and its definition.
+BATCH_PUBLISHER = 'batch_publisher'
```

Comment:
Batch publisher is not a new client, the client is publisher, is just a new widget right? so then should not be declared here, intead should be declared in the qt constants as the assembler widget. Otherwise if it is a pure client it should be created in the client folder.

Reply:
Well spotted, fixing